quickget

```diff
@@ -692,6 +706,29 @@ function get_alma() {
     check_hash "${ISO}" "${HASH}"
     make_vm_config "${ISO}"
 }
+
+function get_alpine() {
+    local HASH=""
+    local ISO=""
+    local URL=""
+    local VERSION=""
+
+    validate_release "releases_alpine"
+
+    if [ "${RELEASE}" == "latest" ]; then
+        VERSION=$(wget -q -O- 'https://alpinelinux.org/releases.json' | jq '.latest_stable' | cut -d "\"" -f 2 | sed 's/v//')
+    else 
+        VERSION="${RELEASE}"
+    fi 
+
+    URL="https://dl-cdn.alpinelinux.org/alpine/v${VERSION}/releases/x86_64"
+    ISO=$(wget -q -O- "https://dl-cdn.alpinelinux.org/alpine/v${VERSION}/releases/x86_64/latest-releases.yaml" | grep -i alpine-standard | grep -i iso: | sed 's/iso://' | sed -r 's/\s+//g')
```

Comment:
We should use the virt edition instead as the kernel is optimised for VM's and less bloated + the user experience is similar to standard edition. Thats what I did in my PR #298